README.md

```diff
@@ -40,21 +40,22 @@
   - [`goal_src/`](#goal_src)
   - [`game` runtime](#game-runtime)
 
-## READ FIRST
+## Please read first
 
-The github repositories are for development of the project and tracking active issues.  Most of the information you will find here pertains to setting up the project for development purposes and is not relevant to a general user.
+Our repositories on GitHub are primarily for development of the project and tracking active issues. Most of the information you will find here pertains to setting up the project for development purposes and is not relevant to the end-user. The Releases page will provide you with a pre-compiled set of tools you need to get this project running on your machine. 
 
-There is a discord server where discussion occur https://discord.gg/VZbXMHXzWv
+For questions or additional information pertaining to the project, we have a Discord for discussion here: https://discord.gg/VZbXMHXzWv
 
-Additionally you can find further information and answers to **frequently asked questions** on the project's main website https://open-goal.github.io/
+Additionally, you can find further documentation and answers to **frequently asked questions** on the project's main website: https://open-goal.github.io/
```

Comment:
old link, we can use our custom domain now - https://opengoal.dev/

Reply:
perfect, i also updated the "Documentation Here" badge to link to the documentation portion of our new domain.